tox.ini

```diff
@@ -5,5 +5,5 @@ skipsdist = True
 
 [testenv:sanity]
 # used just for validating the structure of the test case files themselves
-deps = jsonschema==3.2.0
+deps = jsonschema==4.6.1
```

Comment:
(For reference later, this is the only line that really matters, it updates to a newer version. All the rest is just fixing a deprecation warning).